web-ui/partials/index/panel-orchestration.html

```
@@ -0,0 +1,258 @@
+                                <div v-for="node in taskOrchestrationSelectedRunNodes" :key="node.id" class="task-node-card task-node-card-detail">
+                                    <div class="task-node-head">
+                                        <div>
+                                            <div class="task-node-title">{{ node.title || node.id }}</div>
+                                            <div class="task-node-meta">{{ node.id }} · attempts {{ node.attemptCount || 0 }} · auto-fix {{ node.autoFixRounds || 0 }}</div>
+                                        </div>
+                                        <span :class="['pill', taskRunStatusTone(node.status)]">{{ node.status }}</span>
+                                    </div>
+                                    <div v-if="node.summary" class="task-runtime-item-summary">{{ node.summary }}</div>
+                                    <div v-if="node.error && node.error !== node.summary" class="task-node-deps">错误：{{ node.error }}</div>
+                                    <div class="task-node-deps">依赖：{{ formatTaskNodeDependencies(node) }}</div>
+                                    <pre class="task-log-block">{{ Array.isArray(node.logs) && node.logs.length ? node.logs.map((item) => `${item.at || ''} ${item.level || ''} ${item.message || ''}`.trim()).join('\n') : '(no logs)' }}</pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)